sno/ogr_util.py

```diff
@@ -0,0 +1,23 @@
+from osgeo import ogr
+
+
+def adapt_value_noop(value):
+    return value
+
+
+def adapt_ogr_datetime(value):
+    # OGR uses this strange format: '2012/07/09 09:01:52+00'
+    # We convert back to a normal ISO8601 format.
+    return value.replace('/', '-').replace(' ', 'T').replace('+00', 'Z')
```

Comment:
don't we want to adapt it to a Python datetime?

Reply:
I don't think so - the output of this ends up being passed directly to `msgpack.packb` (in [import_iter_feature_blobs](https://github.com/koordinates/sno/blob/ogr-import/sno/structure.py#L953-L954)), which can't handle datetimes

Reply:
Should probably have it as a unix timestamp in that case? Strings are yuck.

Add to the data-structure-v3 doc?

Reply:
Personally I don't have a problem with strings, doesn't seem worth changing.

If we change it to some other format, we need to convert it when checking out a working copy - GPKG mandates the storage of datetimes as ISO8601 strings:  http://www.geopackage.org/spec/#table_column_data_types